app/content/blog/wiki/allagan-tools-inventory-analysis.md

```
@@ -0,0 +1,93 @@
+
+## How to use the Allgan Data import
+
+1. Make sure you have Dalamund Plugins and Allagan tools installed and that you have the following columns enabled.
+
+![image](https://user-images.githubusercontent.com/17516896/233134089-43ac28d4-a66a-45b1-a82c-73fc504e658f.png)
+
+Make sure source, location, quantity and type are enabled
+
+![image](https://user-images.githubusercontent.com/17516896/235263844-cb4916c5-335e-466e-84bd-04bf112f6bac.png)
+
+2. On the top left menu click > Edit > Copy List Contents > JSON Format
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)